libraries/AP_HAL/utility/RingBuffer.h

```diff
@@ -114,12 +114,12 @@ class ObjectBuffer {
         buffer->clear();
     }
 
-    // return number of objects available to be read
+    // return number of objects available to be read from the front of the queue
     uint32_t available(void) const {
         return buffer->available() / sizeof(T);
     }
 
-    // return number of objects that could be written
+    // return number of objects that could be written to the front of the queue
```

Comment:
All pushes are to the back of the queue.

Reply:
Fixed thanks